deploy/kubernetes/charts/bitcoin/templates/_helpers.tpl

```diff
@@ -0,0 +1,42 @@
+{{- define "bitcoin.name" -}}
+{{- default .Chart.Name .Values.nameOverride | trunc 63 | trimSuffix "-" }}
+{{- end }}
+
+{{- define "bitcoin.fullname" -}}
+{{- if .Values.fullnameOverride }}
+{{- .Values.fullnameOverride | trunc 63 | trimSuffix "-" }}
+{{- else }}
+{{- $name := default .Chart.Name .Values.nameOverride }}
+{{- if contains $name .Release.Name }}
+{{- .Release.Name | trunc 63 | trimSuffix "-" }}
+{{- else }}
+{{- printf "%s-%s" .Release.Name $name | trunc 63 | trimSuffix "-" }}
+{{- end }}
```

Comment:
Can we indent this (file) or will it cause a template failure? Or we can, but it's against convention? Asking due to the complete lack of indents and three `end` statements in a row.

Reply:
For tpl files, there are certain limitation of indentation. When using yaml, you have to be extra cautious with new lines and indentation. These values will be used in yaml files. Therefore, it is better to keep it in this way.

Reply:
Thanks for the answer :)